### PR TITLE
Update social_contract.txt

### PR DESCRIPTION
**Pull request by gu92maak**
please provide feedback on the changes th the social contract I have suggested.
If you don't have feedback, please state so.

### CODE REVIEW

File: project_documentation/social_contract.txt [hunk 1]
```diff
@@ -13,10 +13,52 @@ Social Contract
 
 * Unit testing strategy: TBD
 
+
+* we will have sprinte review and retrospective at 13.00-14.00 on every comming friday
```
Comment (gu92maak) [inline project_documentation/social_contract.txt:17]: What do you guys think about this time?

Comment (antonsvaren) [reply project_documentation/social_contract.txt:17]: Fantastic

File: project_documentation/social_contract.txt [hunk 2]
```diff
@@ -13,10 +13,52 @@ Social Contract
...
 Tools:
 * version control: GitHub 
 * Sprintplanning: Trello
-* IDE: as preferred
+* IDE: as preferred, but  mainly Android Studio
 * Linter: TBD
 * communication and meetings: Discord
-* Language: likely candidates Python and/or Java. But depending on project.
\ No newline at end of file
+* Language: likely candidates Kotlin, Python and/or Java
```
Comment (antonsvaren) [inline project_documentation/social_contract.txt]: Didn't we decide of not having python?

File: project_documentation/social_contract.txt (hunk 1 above)
Comment (eliascalzone) [reply project_documentation/social_contract.txt:17]: good

**Comment by gu92maak** [comment]
Do we want to add Review Rules for Pullrequest for Social Contract?
Do we want to have Social contract rules for when to branch?

File: project_documentation/social_contract.txt (hunk 2 above)
Comment (gu92maak) [reply project_documentation/social_contract.txt]: so Kotlin and Java then, right?

Comment (antonsvaren) [reply project_documentation/social_contract.txt]: Yes :)

Comment (gu92maak) [reply project_documentation/social_contract.txt]: updated in commit 2
Im not sure how to make that reflect here though.

In the "Files changed" tab, i can choose "Changes from all commits" and see the removal of Python.
image

Comment (gu92maak) [reply project_documentation/social_contract.txt]: @antonsvaren If you are happy with my change in commit 2, please do Resolve conversation

File: project_documentation/social_contract.txt (hunk 1 above)
Comment (irenecanovaas) [reply project_documentation/social_contract.txt:17]: for me it is fine

**Comment by antonsvaren** [comment]
I have looked through everything and something that I want to add, is that we should comment our code in order to make it easier for others to understand what we have done.

**Comment by Hannapham1007** [comment]
The contract update looks good.

**Comment by gu92maak** [comment]
In response to Anton https://github.com/antonsvaren/DAT257-DIT257-MANDALORE/pull/2#issuecomment-1253357396

I choose to ignore this in thie PullRequest and I have added it to a new task instead. 
https://trello.com/c/Sgi6SyNY
Is that Ok @antonsvaren? (thumbs up for  Yes)

File: project_documentation/social_contract.txt (hunk 1 above)
Comment (eliascalzone) [reply project_documentation/social_contract.txt:17]: Works

**Review by eliascalzone — APPROVED** [review]

**Pull request merged**